scripts/docker-compose/imports/openzaak-database/database/3-setup-indienen-aansprakelijkstelling-zaaktype.sql

```diff
@@ -174,20 +175,20 @@ VALUES
 (
   (SELECT COALESCE(MAX(id),0) FROM catalogi_resultaattype) + 1, -- Adjust ID as needed
   '00da48aa-9263-4053-bd88-4b9037c9d966', -- UUID
-  'Afgebroken', -- Omschrijving
-  'https://selectielijst.openzaak.nl/api/v1/resultaattypeomschrijvingen/ce8cf476-0b59-496f-8eee-957a7c6e2506', -- Resultaattypeomschrijving
-  'Afgebroken', -- Omschrijving Generiek
-  'https://selectielijst.openzaak.nl/api/v1/resultaten/3db49761-7544-4836-a133-3c77db280e90', -- Selectielijstklasse
-  'blijvend_bewaren', -- Archiefnominatie
-  'P1Y', -- Archiefactietermijn
-  'afgehandeld', -- Brondatum Archiefprocedure Afleidingswijze
-  '', -- Brondatum Archiefprocedure Datumkenmerk
-  false, -- Brondatum Archiefprocedure Einddatum Bekend
-  '', -- Brondatum Archiefprocedure Objecttype
```

Comment:
These comments are not helpful?

Reply:
These are present as part of the schema above, IntelliJ hints and comments. Decided to remove the comments as we usually work with IntelliJ for backend + we rarely change the content of these files.

Reply:
![image](https://github.com/user-attachments/assets/f050a29e-c914-4b13-b2fd-2d25a36cd843)